cmd/cel-shed/header.go

```diff
@@ -30,7 +31,7 @@ Custom store path is not supported yet.`,
 			return fmt.Errorf("not enough arguments")
 		}
 
-		tp := node.ParseType(args[0])
+		tp := nodeconf.ParseType(args[0])
```

Comment:
We should still call the `node/node` one as `node`, while `node` one can be something as `nodebuild` or `nodeasm` or `nodeconf`

Reply:
The former defines every type/config, etc for the node. The latter is node assembler/builder/configurator

Reply:
done